src/objecttypes/setup_configuration/steps/token_auth.py

```diff
@@ -0,0 +1,71 @@
+import logging
+
+from django.core.exceptions import ValidationError
+from django.db import IntegrityError
+
+from django_setup_configuration.configuration import BaseConfigurationStep
+from django_setup_configuration.exceptions import ConfigurationRunFailed
+
+from objecttypes.setup_configuration.models.token_auth import (
+    TokenAuthGroupConfigurationModel,
+)
+from objecttypes.token.models import TokenAuth
+
+logger = logging.getLogger(__name__)
+
+
+class TokenAuthConfigurationStep(
+    BaseConfigurationStep[TokenAuthGroupConfigurationModel]
+):
+    """
+    Configure tokens for other applications to access Objecttypes API
+    """
+
+    namespace = "tokenauth"
+    enable_setting = "tokenauth_config_enable"
+
+    verbose_name = "Configuration to set up authentication tokens for ObjectTypes"
+    config_model = TokenAuthGroupConfigurationModel
+
+    def execute(self, model: TokenAuthGroupConfigurationModel) -> None:
+        for item in model.items:
```

Comment:
I totally agree with you. Fortunately, this exception is already caught by `django-setup-configuration` by default, because the model expects a list, not None.
Librabry interprets it as an `input_value=None` and not as an empty list:

`Input should be a valid list [type=list_type, input_value=None, input_type=NoneType]`

This is the default exception:
```
Unable to satisfy prerequisites for step: Configuration to set up authentication tokens for ObjectTypes:
CommandError: Prerequisites for configuration are not fulfilled: Configuration to set up authentication tokens for ObjectTypes: Failed to load config model for Configuration to set up authentication tokens for ObjectTypes. Further details:
1 validation error for ConfigSettingsSourceTokenauth
tokenauth.items
  Input should be a valid list [type=list_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.10/v/list_type
```